samples/matter/light_switch/VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 2
+VERSION_MINOR = 6
```

Comment:
@LuDuda should it be 2.6.0 or 2.5.99?

Reply:
I see it's `2.5.0` for other applications.. i would use 2.5.99 as it is done in main `VERSION` file.